tests/unit/conftest.py

```diff
@@ -172,20 +172,26 @@ def set_cycling_type(monkeypatch: pytest.MonkeyPatch):
             custom time zone to use.
         dump_format: If using ISO8601, specify custom dump format.
     """
+
     def _set_cycling_type(
         ctype: str = INTEGER_CYCLING_TYPE,
-        time_zone: Optional[str] = None,
+        time_zone: Optional[str] = 'Z',
```

Comment:
What was the reason for this?

Reply:
I can't remember, justify it, or need it to make the tests work. Have reverted.